src/CookiePhysicsComponent.cpp

```diff
@@ -73,11 +74,45 @@ void CookiePhysicsComponent::updatePhysics(float deltaTime) {
         // The color change has to be done here,
         // due to the bouncing off, the cookie never really hits the object.
         // TODO(nurgan) implement checking for object type and only change color if it is a "house"
-        if(objHit->getRenderComponent()->getShader().compare("Green")) {
+        if( objHit->getRenderComponent()->getShader().compare("Green")) {
             objHit->changeShader("Green");
+            cookieState.hits++;
```

Comment:
Oh, whoops, sorry didn't even see the check there, yeah that's probably fine for now, but an actual status will be needed eventually.